From 44bd5b1ccc9a94c9bd3942a1505cccc9b114cb44 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maciej=20Kr=C3=BCger?= <mkg20001@gmail.com>
Date: Thu, 8 Dec 2022 22:12:12 +0100
Subject: [PATCH] prune opkg

---
 include/image.mk | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/include/image.mk b/include/image.mk
index c7a2e082ab..59db63ab8e 100644
--- a/include/image.mk
+++ b/include/image.mk
@@ -330,6 +330,14 @@ target-dir-%: FORCE
 	-$(CP) -T $(mkfs_cur_target_dir).opkg/ $(mkfs_cur_target_dir)/etc/opkg/
 	rm -rf $(mkfs_cur_target_dir).opkg $(mkfs_cur_target_dir).conf
 	$(call prepare_rootfs,$(mkfs_cur_target_dir),$(TOPDIR)/files)
+ifneq ($(GLUON_DEBUG),1)
+ifneq ($(KEEP_OPKG),1)
+		rm -rf $(mkfs_cur_target_dir)/usr/lib/opkg
+		rm -rf $(mkfs_cur_target_dir)/bin/opkg
+		rm -rf $(mkfs_cur_target_dir)/usr/sbin/opkg-key
+		rm -rf $(mkfs_cur_target_dir)/etc/opkg
+endif
+endif
 
 $(KDIR)/root.%: kernel_prepare
 	$(call Image/mkfs/$(word 1,$(target_params)),$(target_params))
-- 
2.38.1

